lib/helpers/lib/each-openapi-module.js

```diff
@@ -163,8 +163,9 @@ function iterate_properties (node, handleLeaf, handleObject, handleArray) {
             ) {
                 node.type = 'object'
             } else {
+                node.type = 'object' // default to object?
```

Comment:
Simplify this section to be:

```
if (!node.type) {
       node.type = 'object'
}
```

Suggested inline comments:

```
// If type is not specified on a schema object, assume it is a n object. This behavior seems to match redocly behavior.
// It is necessary to explicitly set type on object since format_type function will examine the type attribute on the node object.
```

I have some bias for my name not being mentioned in code, but I would accept it, but I think this whole if block can now be simplified.